lib/resources/sessions.js

```diff
@@ -7,12 +7,22 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
+const { DateTime } = require('luxon');
 const Problem = require('../util/problem');
 const { isBlank } = require('../util/util');
 const { verifyPassword } = require('../util/crypto');
 const { endpoint } = require('../http/endpoint');
 const { getOrReject } = require('../util/promise');
-const { success } = require('../util/http');
+const { success, withCookie } = require('../util/http');
+
+const withSessionCookie = withCookie('__Host-session');
```

Comment:
This is because I'm a dummy and having read exactly the same article, and thought to myself, "I should use Secure instead of Host," went ahead and wrote down "Host."

Reply:
Thinking more about it, I think we actually do want `Path=/`. Otherwise, the cookie that's returned for `POST /sessions` won't be sent for `GET /sessions/restore`. If we do want `Path=/`, then I think the `__Host-` prefix sounds good.

Reply:
If no path is given, the cookie is always sent.